shared_types/scamCategory.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+enum scamCategories {
+  phishing = "Phishing",
+  job = "Fake Job Offers",
+}
+
+export const scamCategory = z.nativeEnum(scamCategories);
```

Comment:
Zod says it prefers regular enums to nativeEnums, maybe we should use that instead?

https://zod.dev/README?id=native-enums

E.g.

```typescript
const FishEnum = z.enum(["Salmon", "Tuna", "Trout"]);
```

Reply:
I don't really know... I used nativeEnum before without any issue. With that you can have an ID and a string longer so that when you call ScamCategory you can just write ScamCategory.job and have "Fake Job Offers"